src/build/buildController.ts

```diff
@@ -111,9 +111,12 @@ export class BuildController {
     }
 
     private async validateUserCredential(credential: Credential) {
-        if (credential.signInStatus !== 'SignedIn') {
+        if (this._environmentController.userType === UserType.PublicContributor) {
             return;
         }
+        if (credential.signInStatus !== 'SignedIn' && this._environmentController.userType === UserType.InternalEmployee) {
+            throw new DocsError(`Please sign in first`, ErrorCode.SignedOutInternalUserBuild);   

```

Comment:
Remove the suffixed space. you can check the `Format on save` option in the VSCode, it will format your code when you save it.

Reply:
done, also checked the setting.